news/2024/2024-03-19-bespoke-music-in-osu.md

```diff
@@ -0,0 +1,259 @@
+> 
+> I think this process works very well as we also have a general channel for all our composers to mingle and share ideas for each other's works, allowing for more constructive opinions to be provided throughout the process by equally skilled individuals.
+
+---
+
+The Malaysian tournaments collectively have 19 original tracks in total at the moment, but word is on the street that more is on the way!
+
+## Inside the music pipeline
+
+While [mangomizer](https://osu.ppy.sh/users/1893718) and [pishifat](https://osu.ppy.sh/users/3178418) touched on the process of World Cup music production, how does it fare for other tournaments? **Kasumi** had this to say:
+
+> **Kasumi-sama:** The production process in GTS is very largely hands-off from us as admins. It all depends on the slot we shoot for, but for tiebreakers for example, we usually just give a general rundown of the structure we'd like, as well as a specific atmosphere we're shooting for. At that point, everything like genre, BPM and length, are basically decided by the artist's wish.
+> 
+> We only ask for relatively frequent drafts, in intervals of 30 seconds to 1 minute, so we can check if we first like the general vibe, as one would expect, and if we can map it accordingly to the planned difficulty, of course.
+
+The other tournament hosts interviewed here said basically the same thing, which means there's only one thing to do. (To ask some artists.)
+
+[![](/wiki/shared/news/2024-03-19-bespoke-music-in-osu/takehirotei.png)](https://osu.ppy.sh/users/11793794)
+
+> Hey there, I'm [takehirotei](https://osu.ppy.sh/users/11793794), just your normal rhythm gamer that composes music as a side job. If you're from osu!, you might have heard of me from some tiebreaker songs from some tourneys such as osu! (and taiko) Malaysia Tournaments, [EGTS](/wiki/Tournaments/GTS#expert-global-taiko-showdown), or [4DMWC 2023](/wiki/Tournaments/4DM/2023) and many more, or if you're outside osu!, you might even heard me as the anime song remix guy for remixing some popular titles such as *Idol* (under the Imaginary Reality alias) or *Guitar*, *Loneliness* and *Blue Planet*.
+
+[**takehirotei**](https://osu.ppy.sh/users/11793794) has produced over 20 songs for various community tournaments, 6 of them being for Global Taiko Showdown. Here's their perspective on how it all goes down:
+
+> For the production process, the host will set a deadline on the song, as well as some restrictions on the song (e.g. BPM limit, intensity, genres and more) in order to make sure the song will fit nicely to the planned (or not planned) slots. From time to time, we sometimes also send wips of the current state of the song to make sure that the song is going to the correct direction. This will ensures that we dont have to do unnessary changes later on when the full song is done, which can saves us a lot of time.
```

Comment:
```suggestion
> For the production process, the host will set a deadline on the song, as well as some restrictions on the song (e.g. BPM limit, intensity, genres and more) in order to make sure the song will fit nicely to the planned (or not planned) slots. From time to time, we sometimes also send wips of the current state of the song to make sure that the song is going to the correct direction. This will ensure that we dont have to do unnecessary changes later on when the full song is done, which can saves us a lot of time.
```